docs/man_pages/screenbuilder/upgrade-screenbuilder.md

```diff
@@ -0,0 +1,29 @@
+upgrade-screenbuilder
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder upgrade-screenbuilder`
+
+Upgrades a project to the latest Screen Builder version. This operation regenerates the project and you might lose custom code changes.<% if(isHtml) { %> For more information how to preserve your custom code changes, see [How To Add And Keep Custom Code Changes In Your App](http://docs.telerik.com/platform/screenbuilder/troubleshooting/how-to-keep-custom-code-changes).<% } %>
+
+<% if(isConsole) { %>WARNING: This command is applicable only to Apache Cordova projects created with Screen Builder.<% } %>
+
+<% if(isHtml) { %>
+### Command Limitations
+
+* You can run this command only on Apache Cordova projects created with Screen Builder version 0.0.7 and later. A manual upgrade is required for projects created with an earlier Screen Builder version.
```

Comment:
drop "version" in the first sentence.

For projects created with earlier versions of Screen Builder, you need to upgrade your project manually. You need to create a new Screen Builder project and transfer custom code and resources to the newly created project.